x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/50_semantic_text_query_inference_endpoint_changes.yml

```diff
@@ -78,6 +78,9 @@ setup:
         refresh: true
 ---
 "sparse_embedding changed to text_embedding":
```

Comment:
@carlosdelest, I am having trouble understanding what these tests are doing and why they are failing. The error message they were giving was:
```     
   Caused by:
        java.lang.AssertionError: expected [2xx] status code but api [inference.delete] returned [409 Conflict] [{"error":{"root_cause":[{"type":"status_exception","reason":" Inference endpoint sparse-inference-id is being used in the mapping for indexes: [test-sparse-index]. Ensure that no index mappings are using this inference endpoint, or use force to ignore this warning and delete the inference endpoint.",
```
I am not sure why these tests are attempting to delete an inference endpoint at the begging, should that be happening at the end the test where the endpoint was created?

Reply:
I wrote this test, it's testing when the user replaces an inference endpoint that uses a sparse embedding model with one that uses a text embedding model. We should be able to fix it by using `force` with the `inference.delete` call.

Reply:
thanks Mike!